safe_transaction_service/history/tests/test_erc20_events_indexer.py

```diff
@@ -76,8 +76,22 @@ def test_erc20_events_indexer(self):
             event["args"]["tokenId"] = event["args"].pop("value")
             original_event = copy.deepcopy(event)
             event["args"]["unknown"] = event["args"].pop("tokenId")
-            event["args"]["unknown"] = original_event["args"]["tokenId"]
 
             self.assertEqual(
                 erc20_events_indexer._process_decoded_element(event), original_event
             )
+
+        event = self.ethereum_client.erc20.get_total_transfer_history(
+            from_block=block_number, to_block=block_number
+        )[0]
```

Comment:
Not sure if this is the sort of case you need to `pop` instead of using `index` access to the array 🤔 Just pointing it out :)

Reply:
I could use that, but I don't need to modify the list, only access the element